src/components/announcer/CyverseAnnouncer.js

```diff
@@ -74,3 +70,9 @@ export default CyverseAnnouncer;
 CyverseAnnouncer.propTypes = {
   message: PropTypes.string,
 };
+
+const announce = (msg) => {
+  msgQueue.push(msg);
```

Comment:
👍 . I had this function in `CyverseAnnouncer` as well. Just wanted to try if it make a difference if I made it as `default` export.